install.sh

```diff
@@ -204,6 +204,20 @@ set_current_context_k8s() {
   fi
 }
 
+resume_bamboo_server() {
```

Comment:
This should be called only when we are importing a dataset (the variable is set in config).

Reply:
considered in change above